scripts/check-i18n-links

```diff
@@ -3,22 +3,22 @@
 /**
  * This script reads and parses all of the source code files, reads the US English
  * translation files, extracts all http/s links and checks for broken links.
- * 
+ *
  * This script is used in the PR gate to check for broken links.
  *
  */
 
 const fs = require('fs');
 const path = require('path');
 const yaml = require('js-yaml');
-const request = require('request-promise-native');
```

Comment:
`request-promise-naative` has been deprecated, we can rely on axios instead[^1].

[^1]: https://github.com/request/request/issues/3142